controllers/openstackbaremetalset_controller.go

```diff
@@ -343,7 +344,7 @@ func (r *OpenStackBaremetalSetReconciler) Reconcile(ctx context.Context, req ctr
 			&hostStatus,
 		)
 		if err != nil {
-			return ctrl.Result{RequeueAfter: 10 * time.Second}, err
+			return ctrl.Result{RequeueAfter: 10 * time.Second}, nil
```

Comment:
Is the `err` removed here because including it prevents the requeue after 10 seconds?

Reply:
I just removed it to make the log look cleaner and log an info message instead of an error message in that case, if you all agree :)

Reply:
Sounds good to me!